src/main/java/com/spotinst/sdkjava/model/bl/ocean/ecs/ClusterComputeConfiguration.java

```diff
@@ -11,8 +10,9 @@
     @JsonIgnore
     private Set<String>                       isSet;
     private List<String>                      subnetIds;
-    private ClusterOptimizeImageConfiguration optimizeImage;
+    private ClusterOptimizeImageConfiguration optimizeImages;
     private ClusterLaunchSpecification        launchSpecification;
+    private ClusterInstanceTypes              instanceTypes;
```

Comment:
Didn't find this parameter in the API specification.

Reply:
Parameters keyPair and Instance Types are available in JSON when we check 'view configuration' for Cluster on UI. So Added it. 
So As Discussed, Currently aligning with postman and removing those parameters.